lib/sinon/call.js

```diff
@@ -2,7 +2,7 @@
 
 var arrayProto = require("@sinonjs/commons").prototypes.array;
 var sinonMatch = require("./match");
```

Comment:
This name is incorrect now. It's not `sinonMatch` any longer. It's coming from `samsam`. Also why not be explicit:

```js
var match = require("@sinonjs/samsam").createMatcher;
```